src/workflows/pyfr/components/steps/Simulation/Start/index.js

```diff
@@ -50,13 +52,13 @@ const SimulationStart = React.createClass({
     if (!iniFile) {
```

Comment:
if there is no simululation ini file, get it from the project and copy to to the simulation. `client.copyFile` was missing from the network library here.